src/main/java/com/stories/controller/StoriesController.java

```diff
@@ -90,6 +90,16 @@ public void deleteTask(@Valid @PathVariable String id, @PathVariable String task
 	public StoryDomain updateStory(@Valid @RequestBody StoryDomain request, @PathVariable String id) throws Exception {
 		return storyService.updateStory(request, id);
 	}
+	
+	@ApiOperation(value = " PUT Task ", notes = "This operation will update a task from a story")
+	@ApiResponses({ @ApiResponse(code = 200, message = " Success operation "),
+					@ApiResponse(code = 404, message = " Task not found ") })
+	@ResponseStatus(value = HttpStatus.OK)
+	@PutMapping(value = "/{storyId}/tasks/{taskId}", produces = "application/json")
+	@ResponseBody
+	public TasksDomain updateTask(@Valid @RequestBody TasksDomain task, @PathVariable("storyId") String id, @PathVariable("taskId") String _id) throws Exception{
```

Comment:
Please, verify any unnecessary blank space between the code lines and any unnecessary imports on this class please.

Reply:
Solved